openwisp_firmware_upgrader/tests/test_models.py

```diff
@@ -483,3 +484,21 @@ def test_delete_firmware_image_file(self):
             file_storage_backend.delete(file_name)
             self.assertNotEqual(image.file, None)
             image.delete()
+
+    def test_long_filename_upload(self):
```

Comment:
We need two sub-tests - one would check the filename is within the limits and the other one would check the other way.